src/VirtualMonoRepo/source-mappings.json

```diff
@@ -112,7 +112,8 @@
         },
         {
             "name": "scenario-tests",
-            "defaultRemote": "https://github.com/dotnet/scenario-tests"
+            "defaultRemote": "https://github.com/dotnet/scenario-tests",
+            "disableSynchronization": true
```

Comment:
Is this a point in time setting that will eventually go away once the repo darc flows are all defined?

Reply:
Yes, this whole file will go away most likely